context.go

```diff
@@ -1160,22 +1160,28 @@ func (c *Context) SetAccepted(formats ...string) {
 /***** GOLANG.ORG/X/NET/CONTEXT *****/
 /************************************/
 
-// Deadline always returns that there is no deadline (ok==false),
-// maybe you want to use Request.Context().Deadline() instead.
+// Deadline returns that there is no deadline (ok==false) when c.Request with no Context.
```

Comment:
I'd say this reads better:
```suggestion
// Deadline returns that there is no deadline (ok==false) when c.Request has no Context.
```
Same in the other two comments.

Reply:
done